src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProjectFactory.cs

```diff
@@ -16,7 +17,7 @@
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS
 {
     [Export(typeof(IWorkspaceProjectContextFactory))]
-    internal partial class CPSProjectFactory : IWorkspaceProjectContextFactory
+    internal partial class CPSProjectFactory : ForegroundThreadAffinitizedObject, IWorkspaceProjectContextFactory
```

Comment:
Oh, I thought it defaulted to true. Uh, did that change? I'm terrified now. My worldview has been shattered.